### DISCUSSION TITLE
Margins for windows in ddepin

### DISCUSSION THREAD

**Discussion by apsyhea** [linuxdeepin/developer-center#3632] [Q&A | 问答板块] [answered]
How can I adjust the margins from the edge for windows in dde in such a way as in the screenshot?

![image](https://user-images.githubusercontent.com/37932705/218319783-63114897-a791-4568-a5da-267e4149165c.png)

**Comment by BLumia** [comment, marked as answer]
https://store.kde.org/p/1619642 this kwin script might works.

cc @justforlxz

**Reply by apsyhea** [reply]
Thanks

**Comment by apsyhea** [comment]
`qdbus: could not find a Qt installation of ''`
